### PR TITLE
[Tag] allow Tag to accept any type of children

### PR DESCRIPTION
**Pull request by ipanasenko**
What was a reason for restricting it? In our code we render translated strings inside `Tag`, like this:

```ts
<Tag id="new-search">
  <FormattedMessage id="empty-state.media-from-wix-no-results-new-search" />
</Tag>
```

### CODE REVIEW

File: src/Tag/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import * as React from 'react';
 
 export interface TagProps {
-  children: string;
```
Comment (itaytay) [inline src/Tag/index.d.ts]: Why remove it and not just replace it with React.ReactNode ?
cc @mosheKerbel

Comment (ipanasenko) [reply src/Tag/index.d.ts]: because it's ReactNode by default

Comment (ipanasenko) [reply src/Tag/index.d.ts]: But it's optional by default, so we need to specify it explicitly as ReactNode if children are required

Comment (mosheKerbel) [reply src/Tag/index.d.ts]: right, children is an exception and by default exist on the type.

**Comment by mosheKerbel** [comment]
> What was a reason for restricting it? In our code we render translated strings inside `Tag`, like this:
> 
> ```ts
> <Tag id="new-search">
>   <FormattedMessage id="empty-state.media-from-wix-no-results-new-search" />
> </Tag>
> ```

you are right, indeed node should be the allowed type.

**Pull request merged**